app/views/campaign_pages/index.slim

```diff
@@ -11,7 +10,7 @@
           th Actions
           th
       tbody
-        - @campaign_pages.each do |page|
+        - [*@campaign_pages].each do |page|
```

Comment:
ahhh clever. it's a way of being able to call `#each` even when `@campaign_pages` is nil. you could just wrap the whole thing in a conditional on that, which would be less concise but definitely clearer... I'm pretty indifferent though I imagine omar might have an opinion about the code style one way or the other.